modules/programs/mako.nix

```
@@ -0,0 +1,264 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.mako;
+
+in {
+  options = {
+    programs.mako = {
+      enable = mkEnableOption "Mako, lightweight notification daemon for Wayland";
+
+      maxVisible = mkOption {
+        default = 5;
+        type = types.nullOr types.int;
+        description = ''
+          Set maximum number of visible notifications. Set -1 to show all.
+        '';
+      };
+
+      sort = mkOption {
+        default = "-time";
+        type = types.nullOr types.str;
+        description = ''
+	  Sorts incoming notifications by time and/or priority in ascending(+) or descending(-( order. E.g.: +/-time, +/-priority
```

A matter of preference, but it'd be nice if you could reflow this to ~80 characters.

```suggestion
	  Sorts incoming notifications by time and/or priority in ascending(+) or descending(-) order. E.g.: +/-time, +/-priority
```

Also, maybe it would be a good idea to make it `types.enum` since we have only a few valid settings.